mesheryctl/internal/cli/root/system/preflight.go

```diff
@@ -0,0 +1,54 @@
+package system
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+
+	meshkitkube "github.com/layer5io/meshkit/utils/kubernetes"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/spf13/cobra"
+)
+
+var preCheckCmd = &cobra.Command{
+	Use:   "pre",
+	Short: "run basic checks to verify environment for deployment",
+	Long:  `To verify environment readiness for a Meshery deloyment `,
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		log.Info("Docker /n--------------")
+		//Check whether docker daemon is running or not
+		if err := exec.Command("docker", "ps").Run(); err != nil {
```

Comment:
Shouldn't we have a first check on whether docker is installed or not (docker --version) ? and any minimum version check for docker version?

Reply:
@navendu-pottekkat Yeah, we should include a `√ is running the minimum docker version`, Thoughts?

Reply:
@hexxdump `Docker is running` is more or less similar to checking whether docker is installed, :thought_balloon:

Reply:
@piyushsingariya, Agree. However, if there is any requirement on minimum version of docker, we need to check for 'docker --version'